ovirt/resource_ovirt_vm.go

```diff
@@ -104,6 +105,16 @@ func resourceOvirtVM(c *providerContext) *schema.Resource {
 				},
 				Description: "in MB",
 			},
+			"guaranteed_memory": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ValidateFunc: validation.IntAtLeast(1),
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					// Suppress diff if new memory is not set
+					return new == "0"
+				},
```

Comment:
Not sure if the DiffSuppressFunc is really needed or not, but we can probably move it to a unified function to stop the duplication

Reply:
yea its needed, in my tests I've had issues without it.

Reply:
moved to unified function